### PR TITLE
Up dependencies

### PR DESCRIPTION
**Pull request by tmathern**
- Up (some) dependencies
- Fix worker tests: needs latest aio cli + latest plugin version (worked locally, at least, when using fixes from https://github.com/adobe/aio-cli-plugin-asset-compute/pull/57 too)

### CODE REVIEW

**Comment by tmathern** [comment]
Needs https://github.com/adobe/aio-cli-plugin-asset-compute/pull/57 (fixes an edge case which flags tests as failed when they are not).

File: package.json [hunk 1]
```diff
@@ -12,49 +12,49 @@
...
     "gm": "^1.23.1",
-    "image-size": "^0.9.3",
+    "image-size": "^1.0.0",
     "jsonwebtoken": "^8.5.1",
-    "mime-types": "^2.1.28",
+    "mime-types": "^2.1.30",
     "openwhisk": "^3.21.3",
     "strip-ansi": "^6.0.0",
-    "valid-data-url": "^3.0.1",
+    "valid-data-url": "^4.0.0",
```
Comment (jdelbick) [inline package.json:33]: were there any breaking changes?

**Review by jdelbick — APPROVED** [review]
conditional approval as long as tests pass

File: package.json (hunk 1 above)
Comment (tmathern) [reply package.json:33]: No, otherwise I would have written it in the description.
